mutable_tree.go

```diff
@@ -1081,6 +1067,12 @@ func (tree *MutableTree) saveNewNodes() error {
 			nonce:   nonce,
 		}
 
+		// Save the root node at InitialVersion to nonce of 0 to hide it from
+		// the version list.
+		if isInitialVersion && nonce == 1 {
+			node.nodeKey.nonce = 0
+		}
```

Comment:
this has no effect on saving the actual 1st version (in block 1) of module state because `isInitialVersion` can't be true when `tree.ndb.opts.InitialVersion == 1`

am i understanding that correctly?

Reply:
Yup! Overridden behavior skips `InitialVersion == 1`